Proof-reading edits :bowtie: (#405)
JoshOrndorff/recipes: text/runtime-printing.md

## No Std

The very first line of code tells the Rust compiler that this crate should not use Rust's standard
library except when explicitly told to. This is useful because Substrate runtimes compile to Web

> Assembly when the standard library is not available.

Reviewer: I think we should use either "WebAssembly" or "Wasm" exclusively.

## Tests

> Following, we see a reference to the tests module. This pallet, as with most recipes pallets, has tests written in a separate file called

Reviewer: I think "next" is a more appropriate word to use here.

Author: Yes, I just found that that word's use is a little repetitive in here.

As you can see, our `hello-substrate` pallet has a dispatchable call that takes a single argument,
called `origin`. The call returns a
[`DispatchResult`](https://substrate.dev/rustdocs/v2.0.0/frame_support/dispatch/type.DispatchResult.html) which

> can be either `Ok(())` indicating that the call succeeded, or an `Err` which is demonstrated in most other pallets from this cookbook.

Reviewer: Nitpick: if we are going to audit this repo for grammatical improvements, I would very much like to catch all improper usages of "which". In short, "which" should always be preceeded by a comma, as in "...or an `Err`, which is demonstrated..." Ref: https://www.1klb.com/posts/2014/01/08/that-vs-which/

Reviewer: With respect to the actual change being made on this line, I'm not entirely sure if I agree with it. I may prefer something like "other Recipe pallets" - I think it is more specific than "from this cookbook".

Author: That works fine!

`-lruntime=debug` when running the kitchen node. So, for the kitchen node, the command would become
`./target/release/kitchen-node --dev -lruntime=debug`.

> The next line uses `debug::info!` macro to log to the screen and to inspect the
> variable's content. The syntax inside the macro is very similar to what the regular Rust macro
> `println!` takes.

Reviewer: ```suggestion
The next line uses the `debug::info!` macro to log the variable's value. The syntax inside the
 `debug::info!` macro is very similar to the syntax used by the
 [`std::println!` macro](https://doc.rust-lang.org/std/macro.println.html).
```

**Runtime logger note:** When we execute the runtime in native, `debug::info!` messages are printed.

> However, if we execute the runtime in WASM, then an additional step to initialise

Reviewer: This is incorrect. The abbreviation is correctly stylized as "Wasm".
